### PR TITLE
[circle-interpreter] Introduce `circle-interpreter` driver

### PR DESCRIPTION
**Pull request by yunjayh**
This commit introduces `circle-interpreter` driver.

Signed-off-by: yunjay-hong <yunjay.hong@samsung.com>

---

for: #9634 
draft: #9635

### CODE REVIEW

File: compiler/circle-interpreter/CMakeLists.txt [hunk 1]
```diff
@@ -0,0 +1,14 @@
+set(INTERPRETER
+      src/CircleInterpreter.cpp
+   )
+
+add_executable(circle-interpreter ${INTERPRETER})
+target_link_libraries(circle-interpreter PRIVATE arser)
+target_link_libraries(circle-interpreter PRIVATE loco)
+target_link_libraries(circle-interpreter PRIVATE luci_import)
+target_link_libraries(circle-interpreter PRIVATE luci_export)
```
Comment (jinevening) [inline compiler/circle-interpreter/CMakeLists.txt:9]: luci_export seems unnecessary.

File: compiler/circle-interpreter/src/CircleInterpreter.cpp [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <arser/arser.h>
+#include <luci/ImporterEx.h>
+#include <luci_interpreter/Interpreter.h>
+#include <luci/CircleExporter.h>
+#include <luci/CircleFileExpContract.h>
```
Comment (jinevening) [inline compiler/circle-interpreter/src/CircleInterpreter.cpp:21]: These headers seem unnecessary.
```suggestion

```

File: compiler/circle-interpreter/src/CircleInterpreter.cpp [hunk 3]
```diff
@@ -0,0 +1,147 @@
...
+
+  try
+  {
+    arser.parse(argc, argv);
+  }
+  catch (const std::runtime_error &err)
+  {
+    std::cout << err.what() << std::endl;
+    std::cout << arser;
+    return 255;
```
Comment (jinevening) [inline compiler/circle-interpreter/src/CircleInterpreter.cpp:98]: For consistency with the below code.

```suggestion
    return EXIT_FAILURE;
```

File: compiler/circle-interpreter/CMakeLists.txt (hunk 1 above)
Comment (yunjayh) [reply compiler/circle-interpreter/CMakeLists.txt:9]:
```suggestion
```

**Review by jinevening — APPROVED** [review]
LGTM

(optional) Not for this PR, but it would be better to add README that briefly explains what this module is (e.g., purpose, how to use, difference from luci_eval_driver).

**Review by seanshpark — APPROVED** [review]
LGTM

**Pull request merged**